src/Service.Tests/Authorization/AuthorizationResolverUnitTests.cs

```diff
@@ -965,10 +970,10 @@ public void ParseValidDbPolicy(string policy, string expectedParsedPolicy)
         [DataRow(ClaimValueTypes.UInteger64,    "18446744073709551615",               true, DisplayName = "ulong")]
         [DataRow(ClaimValueTypes.Double,        "12.34",                              true, DisplayName = "decimal")]
         [DataRow(ClaimValueTypes.Double,        "12.345",                             true, DisplayName = "double")]
-        [DataRow(JsonClaimValueTypes.JsonNull,  "null",                               true, DisplayName = "Json null literal")]
+        [DataRow(Microsoft.IdentityModel.JsonWebTokens.JsonClaimValueTypes.JsonNull,  "null",                               true, DisplayName = "Json null literal")]
```

Comment:
is this required since we already imported it above?

Reply:
thanks for the feedback. updated to not have namespace. confirmed via intellisense.